docs/getting-started-guides/kubeadm.md

```diff
@@ -0,0 +1,245 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+This quickstart shows you how to easily install a secure Kubernetes cluster on machines running Ubuntu 16.04 or CentOS 7.
```

Comment:
we haven't tested them so I'm not going to advertise that we work with them. We can do so once we have tested them.